betterproto/lib/google/protobuf.py

```diff
@@ -0,0 +1,1312 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# sources: google/protobuf/any.proto, google/protobuf/source_context.proto, google/protobuf/type.proto, google/protobuf/api.proto, google/protobuf/descriptor.proto, google/protobuf/duration.proto, google/protobuf/empty.proto, google/protobuf/field_mask.proto, google/protobuf/struct.proto, google/protobuf/timestamp.proto, google/protobuf/wrappers.proto
+# plugin: python-betterproto
+from dataclasses import dataclass
+from typing import Dict, List
+
+import betterproto
+
+
+class Syntax(betterproto.Enum):
+    """The syntax in which a protocol buffer element is defined."""
+
+    # Syntax `proto2`.
+    SYNTAX_PROTO2 = 0
+    # Syntax `proto3`.
+    SYNTAX_PROTO3 = 1
+
+
+class FieldKind(betterproto.Enum):
+    TYPE_UNKNOWN = 0
+    TYPE_DOUBLE = 1
+    TYPE_FLOAT = 2
+    TYPE_INT64 = 3
+    TYPE_UINT64 = 4
+    TYPE_INT32 = 5
+    TYPE_FIXED64 = 6
+    TYPE_FIXED32 = 7
+    TYPE_BOOL = 8
+    TYPE_STRING = 9
+    TYPE_GROUP = 10
+    TYPE_MESSAGE = 11
+    TYPE_BYTES = 12
+    TYPE_UINT32 = 13
+    TYPE_ENUM = 14
+    TYPE_SFIXED32 = 15
+    TYPE_SFIXED64 = 16
+    TYPE_SINT32 = 17
+    TYPE_SINT64 = 18
+
+
+class FieldCardinality(betterproto.Enum):
+    CARDINALITY_UNKNOWN = 0
+    CARDINALITY_OPTIONAL = 1
+    CARDINALITY_REQUIRED = 2
+    CARDINALITY_REPEATED = 3
+
+
+class FieldDescriptorProtoType(betterproto.Enum):
+    TYPE_DOUBLE = 1
+    TYPE_FLOAT = 2
+    TYPE_INT64 = 3
+    TYPE_UINT64 = 4
+    TYPE_INT32 = 5
+    TYPE_FIXED64 = 6
+    TYPE_FIXED32 = 7
+    TYPE_BOOL = 8
+    TYPE_STRING = 9
+    TYPE_GROUP = 10
+    TYPE_MESSAGE = 11
+    TYPE_BYTES = 12
+    TYPE_UINT32 = 13
+    TYPE_ENUM = 14
+    TYPE_SFIXED32 = 15
+    TYPE_SFIXED64 = 16
+    TYPE_SINT32 = 17
+    TYPE_SINT64 = 18
+
+
+class FieldDescriptorProtoLabel(betterproto.Enum):
+    LABEL_OPTIONAL = 1
+    LABEL_REQUIRED = 2
+    LABEL_REPEATED = 3
+
+
+class FileOptionsOptimizeMode(betterproto.Enum):
+    SPEED = 1
+    CODE_SIZE = 2
+    LITE_RUNTIME = 3
+
+
+class FieldOptionsCType(betterproto.Enum):
+    STRING = 0
+    CORD = 1
+    STRING_PIECE = 2
+
+
+class FieldOptionsJSType(betterproto.Enum):
+    JS_NORMAL = 0
+    JS_STRING = 1
+    JS_NUMBER = 2
+
+
+class MethodOptionsIdempotencyLevel(betterproto.Enum):
+    IDEMPOTENCY_UNKNOWN = 0
+    NO_SIDE_EFFECTS = 1
+    IDEMPOTENT = 2
+
+
+class NullValue(betterproto.Enum):
+    """
+    `NullValue` is a singleton enumeration to represent the null value for the
+    `Value` type union.  The JSON representation for `NullValue` is JSON
+    `null`.
+    """
+
+    # Null value.
+    NULL_VALUE = 0
+
+
+@dataclass
+class Any(betterproto.Message):
+    """
+    `Any` contains an arbitrary serialized protocol buffer message along with a
```

Comment:
This makes me think comment handling could be better...

Reply:
👍  Apparently, newlines are stripped. I've made an issue: #80